Bottom: 8e2dd785b6a35f3ab78f71f27382fb6523ae8732
Top:    aa78bec9319affa16300da970e19ed9b4467c584
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-16 09:17:28 -0500

Refresh of use-docker-named-volumes

---
diff --git a/roles/docker-mediawiki/tasks/main.yml b/roles/docker-mediawiki/tasks/main.yml
index 4e7b5c22..1aeb23dd 100644
--- a/roles/docker-mediawiki/tasks/main.yml
+++ b/roles/docker-mediawiki/tasks/main.yml
@@ -151,7 +151,7 @@
     state: started
     volumes:
       - '{{ openssl_dv_name }}:/etc/ssl/private:z'
-      - '{{ wiki_dv_name }}/var/www/html:z'
+      - '{{ wiki_dv_name }}:/var/www/html:z'
 
 # *****************************************************************************
 # restore?
